jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -95,16 +154,76 @@ jobs:
       - image: cimg/go:1.18.1
     parallelism: 4
     resource_class: large
-    working_directory: ~/my-app
     steps:
-      - run: go test -v $(go list ./... | circleci tests split --split-by=timings)
+      - run: go list ./... | circleci tests run --command "xargs gotestsum --junitfile junit.xml --format testname --" --split-by=timings --timings-type=name
 ```
 
 For a more detailed walkthrough, read the [guide to using the CLI to split tests](/docs/use-the-circleci-cli-to-split-tests), or follow our [Test splitting tutorial](/docs/test-splitting-tutorial).
 
 The first time the tests are run there will be no timing data for the command to use, but on subsequent runs the test time will be optimized.
 {: class="alert alert-info"}
 
+### The tests run command
+{: #the-tests-run-command }
+
+Use the `circleci tests run` command to run your tests, split your tests across parallel executors and take advantage of the [rerun failed tests](/docs/rerun-failed-tests/) options.
+
+The following table show a full list of option flags available when using `circleci tests run`.
+
+Flag | Type | Description | Required?
+--- | --- | --- | ---
+`--command` | string | The command string is the script that will be run for a list of tests determined by the options provided to the plugin | Yes
+`--index` | uint | index of node can also be set with CIRCLE_NODE_INDEX. (default 1) | No
+`--split-by` | string | how to weight the split, allowed values are "name", "filesize", and "timings". (default "name") | No
+`--timings-type` | string | name of the field to use from historical test results when matching against the test names given to the command in order to determine their historical timings, previous status and flakiness. Available values: `classname`, `name`, `file` (default). (default "none") | No
```

Comment:
The explanation is confusing me here, also there are two defaults listed 🤔  Think I took this from the CLI help output

Reply:
@sebastian-lerner

Reply:
file is the default for timings type. sorry what else is confusing you? then i can help resolve the confusion

Reply:
Thanks, I think I understand it now, I'll just remove `(default "none")`